go/test/endtoend/sharded/shared_keyspace_test.go

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
Is the file supposed to be called sharded_keyspace_test.go?

Reply:
Done